webtester-core/src/test/java/integration/AbstractWebTesterIntegrationTest.java

```diff
@@ -54,6 +55,10 @@ protected String getHTMLFilePath() {
 
     // UTILITY
 
+    protected void assertPageObjectCanBeInitialized(PageObject object) {
```

Comment:
I looked at some of the branches in your repo (EmailField, UrlField, SearchField) and didn't find any references to the method. Can you point me to a branch, where you are actually using it? I don't want to integrate dead code.